Source/Core/DolphinQt2/Config/ControllersWindow.cpp

```diff
@@ -69,8 +70,9 @@ static int FromWiimoteMenuIndex(const int menudevice)
 ControllersWindow::ControllersWindow(QWidget* parent)
     : QDialog(parent),
       m_configure_icon(Settings().GetThemeDir().append(QStringLiteral("config.png"))),
-      m_gamecube_icon(Settings().GetResourcesDir().append(QStringLiteral("Platform_Gamecube.png"))),
-      m_wii_icon(Settings().GetResourcesDir().append(QStringLiteral("Platform_Wii.png")))
+      m_gamecube_icon(
+          Settings().GetResourcesDir().append(QStringLiteral("Platform_Gamecube@2x.png"))),
+      m_wii_icon(Settings().GetResourcesDir().append(QStringLiteral("Platform_Wii@2x.png")))
```

Comment:
The icon is scaled in such a way that at almost all scales it's appropriate to use a ``@2x`` version